pkg/storage/mysql/mysqlstore.go

```diff
@@ -192,18 +192,12 @@ func (s *sqlDBStore) Put(k string, v []byte) error {
 		return errors.New("key and value are mandatory")
 	}
 
-	// Use query is used to select the created database without this DDL operations are not permitted
-	_, err := s.db.Exec(fmt.Sprintf(useDBQuery, s.dbName))
-	if err != nil {
-		return fmt.Errorf("failed to use db %s: %w", s.dbName, err)
-	}
-
 	// create upsert query to insert the record, checking whether the key is already mapped to a value in the store.
-	createStmt := "INSERT INTO `" + s.tableName + "` VALUES (?, ?) ON DUPLICATE KEY UPDATE value=?"
+	insertStmt := "INSERT INTO " + s.tableName + " VALUES (?, ?) ON DUPLICATE KEY UPDATE value=?"
```

Comment:
Why did you remove ``` character?

Reply:
failure while executing insert statement on table prefixdb_test.prefixdb_test: Error 1046: No database selected.
if i dont remove `character, dot doesnt get picked up so thats why i removed. but now hyphen in the uuid key string doesnt work without `

Reply:
@soluchok backticks are already there in `s.tableName`: https://github.com/hyperledger/aries-framework-go/pull/2194/files#diff-5c03fdd2a42c9a160e4a9b056cb9db20R131